yarn.lock

```diff
@@ -13543,7 +13538,7 @@ request-promise-native@^1.0.5:
     stealthy-require "^1.1.1"
     tough-cookie "^2.3.3"
 
-request@^2.87.0, request@^2.88.0:
```

Comment:
This is changed in #652, need to keep an eye on it so we don't have merge conflicts